doc/manual/installing.rst

```diff
@@ -321,3 +321,23 @@ Other options include:
   - ``ext0_bypass_125`` and ``ext0_bypass_100`` - explicit aliases for ``ext0_bypass``.
 
 Availability of these options depends on the board and their configuration - specific setting may or may not be supported.
+
+Show channel name in RTIO error report
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+First, create a channel names text file. Then write a new line for each channel with format: ``<channel_number>:<channel_name>``. For example, naming channel 0 as led0 can be written as ``0:led0``. Channel name should not contain any ``:``, ``,`` and space.
```

Comment:
This says "new line" but the firmware splits with a ``,`` delimiter. What gives?